geonetwork/core-geonetwork
# add capability to add-to-map for tms protocol

Initial version of an 'add-to-map' functionality for datasets exposed as TMS

Some examples:
http://geodata.nationaalgeoregister.nl/tiles/service/tms/1.0.0/aan/EPSG:28992
https://geodata.nationaalgeoregister.nl/tiles/service/tms/1.0.0/brtachtergrondkaartpastel/EPSG:3857
(there are some challenges with this service, it doesn't support the 'TileMap Resource' operation, maybe this causes the problems below)

To verify:
- TMS has a concept of tilemaps and profiles, should we consider them as 'layer' and 'matrixset (=combination of style and projection)' -> this may require some 'capabilities' parsing, before adding the layer
- I use gnMap.createLayerFromProperties(); to add the layer, however the layer is not added and no warning shown, this requires optimisation in ngeo? Or do I need to check things in the tms-capabilities before calling createLayerFromProperties?

TMS spec here http://wiki.osgeo.org/wiki/Tile_Map_Service_Specification
